pydicom/pixel_data_handlers/gdcm_handler.py

```diff
@@ -16,10 +17,40 @@
 
 
 def supports_transfer_syntax(dicom_dataset):
+    """
+    Returns
+    -------
+    True:
+        if this pixel data handler might support this transfer syntax.
+
+    False:
+        to prevent any attempt to try to use this handler
+        to decode the given transfer syntax
+    """
     return True
 
 
 def get_pixeldata(dicom_dataset):
+    """
+    Use the GDCM package to decode the PixelData attribute
+
+    Returns
+    -------
+    A correctly sized (but not shaped) numpy array of the entire data volume
+
+    Raises
+    ------
+    ImportError:
+        if the required packages are not available
+
+    TypeError:
```

Comment:
No colon after the exception name.

[Numpy docstring conventions](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)